### ISSUE TITLE
Tabs not rendering properly when number of tabs decreases

### ISSUE THREAD

**Issue by Krever** [GeekyAnts/NativeBase#3032]
## I have gone through these following points

- [x] Check latest documentation: https://docs.nativebase.io/
- [x] Check for existing open/closed issues for a possible duplicate before creating a new issue: https://github.com/GeekyAnts/NativeBase/issues
- [x] Use the latest NativeBase release: https://github.com/GeekyAnts/NativeBase/releases
- [x] Check examples from NativeBase KitchenSink https://github.com/GeekyAnts/NativeBase-KitchenSink
- [x] For discussion purpose make use of NativeBase Slack: http://slack.nativebase.io/
- [x] For queries related to theme, check [Theme Variables](https://docs.nativebase.io/docs/ThemeVariables.html) from Docs and live NativeBase Theme Editor http://nativebase.io/customizer/

#### node, npm, react-native, react and native-base version, expo version if used, xcode version

```
    "expo": "^35.0.0",
    "native-base": "^2.13.8",
    "react": "16.8.3",
    "react-native": "https://github.com/expo/react-native/archive/sdk-35.0.0.tar.gz",
```

#### Expected behaviour

When number of tabs decreases so the current tab is no longer present, then Tabs header renders properly picking either first or last tab as the current one. 

#### Actual behaviour

It renders blank tab and works only after manually navigating to existing tab

#### Steps to reproduce

1. Create tabs component with dynamic list of tabs passed as props
2. navigate to last tab
3. invoke action that will decrease number of tabs

Sample component:
```
export interface Props {
  tabs: string[]
}

export const Foo: React.FC<Props> = (props) => {
  return (
    <Container>
      <Tabs renderTabBar={() => <ScrollableTab />}>
        {props.tabs.map((tab, idx) =>
          <Tab heading={tab} key={idx}>
            <Text>XXX</Text>
          </Tab>,
        )}
      </Tabs>
    </Container>
  )
}
```

#### Is the bug present in both iOS and Android or in any one of them?

Checked only on android

**Comment by hanykumar** [comment]
Hi @Krever, I went through your code mentioned. I could not find any issue in iOS. Could you just provide snack for the same for android.

**Comment by Krever** [comment]
Hey @hanykumar, I only checked this on android. What do you mean by snack?

**Comment by hanykumar** [comment]
Hi @Krever , you can try your code on [snack](https://snack.expo.io/). It must be working on your snack account before providing the snack project link to us.

latest version of Native base has been released. The issues should be fixed now.
[Native Base 2.13.12](https://github.com/GeekyAnts/NativeBase/releases/tag/v2.13.12)

**Comment by Krever** [comment]
Thanks. The project in which I stumbled upon this issue is shelved for now but I will try to verify as soon as I'm back to working on it. In the meantime I will close the issue.
